micronota/db/uniref.py

```diff
@@ -89,11 +89,14 @@
 from ..util import _overwrite, _download
 
 
+_status = ['Swiss-Prot', 'TrEMBL']
```

Comment:
yeah. it should be ok. uniprotkb use that. basically the process is they review trembl record and update its status to swiss-sprot